pytket/extensions/qiskit/backends/crosstalk_model.py

```diff
@@ -0,0 +1,378 @@
+                    )
+                power = 1 / (self.N * gt)
+                u_i = self._matrix_power(u, power)
+                for _ in range(round(self.N * gt)):
+                    u_i_box = self._get_ubox(u_i)
+                    self._slices.append(
+                        FractionalUnitary(
+                            Command(u_i_box, cmd.args), round(self.N * gt)
+                        )
+                    )
+
+    def _add_zz_crosstalks(self, noise_slice: Slice) -> None:
+        for (q0, q1), zz in self.ct_params.zz_crosstalks.items():
+            if q0 in self.all_qubits and q1 in self.all_qubits:
+                Z = zz / self.N
+                if abs(Z) > EPS:
+                    noise_slice.append(
+                        NoiseGate(
+                            Command(Op.create(OpType.ZZPhase, Z), [q0, q1]),
+                            "zz_crosstalks",
+                        )
+                    )
+
+    def _add_single_q_phase(self, noise_slice: Slice) -> None:
+        for q, z in self.ct_params.single_q_phase_errors.items():
+            if q in self.all_qubits:
+                Z = z / self.N
+                if abs(Z) > EPS:
+                    noise_slice.append(
+                        NoiseGate(
+                            Command(Op.create(OpType.Rz, Z), [q]), "single_q_phase"
+                        )
+                    )
+
+    def _add_two_q_induced_phase(
+        self, unitary_slice: Slice, noise_slice: Slice
+    ) -> None:
+        for inst in unitary_slice:
+            if (
+                isinstance(inst, FractionalUnitary)
+                and inst.cmd.op.type == OpType.Unitary2qBox
+            ):
+                qubits = inst.cmd.qubits
+                value = self.ct_params.two_q_induced_phase_errors.get(
+                    (qubits[0], qubits[1])
+                )
+                if value is None:
+                    raise ValueError(
+                        f"two_q_induced_phase_errors does not have: {qubits}"
+                    )
+                Z = value[1] / inst.n_fractions
+                if abs(Z) > EPS:
+                    noise_slice.append(
+                        NoiseGate(
+                            Command(Op.create(OpType.Rz, Z), [value[0]]),
+                            "two_q_induced_phase",
+                        )
+                    )
+
+    def _add_non_markovian(self, noise_slice: Slice) -> None:
+        for q, zx, zz in self.ct_params.non_markovian_noise:
+            two_level_q = self.two_level_map[q]
+            ZX = zx / self.N
+            ZZ = zz / self.N
+            if abs(ZZ) > EPS:
+                noise_slice.append(
+                    NoiseGate(
+                        Command(Op.create(OpType.ZZPhase, ZZ), [q, two_level_q]),
+                        "non_markovian",
+                    )
+                )
+            if abs(ZX) > EPS:
+                RZX = Unitary2qBox(
```

Comment:
In the current setup, `Unitary2qBox` and `Unitary1qBox` in the Circuit slices require amplitude damping and phase damping, via a Qiskit NoiseModel`, while other gates added for other noise, such as the given non-Markovian noise, don't.

At a later date maybe we _should_ update our NoiseModel conversion to label things, but for now this constraint works for us. The only thing is this RZX is added as a `Unitary2qBox` so would have the same noise added to it, which we don't want.

There is a simple fix though, which is to do:
```
RZX = PauliExpBox([Pauli.Z, Pauli.X], ZX * math.pi/2)
```
Instead (though confirming the angle is correct, I may have written it wrong).